### PR TITLE
Selenium test with qdemo

### PR DESCRIPTION
**Pull request by ankitjavalkar**
Please merge only **after** merging #127

### CODE REVIEW

File: yaksh/bash_files/sample2.args [hunk 1]
```diff
@@ -0,0 +1 @@
+file1.csv file2.csv file3.csv
```
Comment (prabhuramachandran) [inline yaksh/bash_files/sample2.args]: Why are these needed?

File: yaksh/bash_files/sample2.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+cat $1 | cut -d: -f2 | paste -d: $3 - $2
```
Comment (prabhuramachandran) [inline yaksh/bash_files/sample2.sh]: Why is this needed?  I thought the file upload allowed us to do this nicely?

File: yaksh/tests/load_test.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        code_server_pool = ServerPool(ports=settings.SERVER_PORTS, pool_port=settings.SERVER_POOL_PORT)
+        cls.code_server_pool = code_server_pool
+        cls.code_server_thread = t = Thread(target=code_server_pool.run)
+        t.start()
+
+        demo_course_setup = create_demo_course()
+
+    @classmethod
+    def tearDownClass(cls):
+        # User.objects.all().delete()
```
Comment (prabhuramachandran) [inline yaksh/tests/load_test.py]: Why is there commented code here?

File: .travis.yml [hunk 4]
```diff
@@ -14,11 +14,16 @@ install:
...
   - sudo apt-get update -qq
   - sudo apt-get install -y scilab
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
 
 # command to run tests
 script:
   - python manage.py test -v 2 yaksh
+  - python manage.py test -v 2 yaksh.tests.load_test
```
Comment (prabhuramachandran) [inline .travis.yml]: Why is this not run along with the rest?  Given the name of the class, won't it automatically be picked up?  At least a comment at a relevant location for this would be important.

File: yaksh/bash_files/sample2.sh (hunk 2 above)
Comment (adityacp) [inline yaksh/bash_files/sample2.sh]: This sample script is used in evaluator to check user answer. 
This script acts as test case for bash question.
It is not related to file uploads.
same for `sample2.args`

File: yaksh/tests/load_test.py (hunk 3 above)
Comment (ankitjavalkar) [inline yaksh/tests/load_test.py]: Removed

File: .travis.yml (hunk 4 above)
Comment (ankitjavalkar) [inline .travis.yml]: The load_test.py uses a different type of TestCase class called LiveServerTestCase. The unit tests use the usual TestCase class.
The LiveServerTestCase class deals with Tear down a little differently - It only removes the rows of the table and does not reset the Primary keys. Because of this the subsequent model test cases fail.
Everything runs fine if both model test cases and selenium load test are run seperately.

The situation that I faced has been described in the question below. I tried a few solution but nothing seems to fix the issue
http://stackoverflow.com/questions/32998166/with-py-test-database-is-not-reset-after-liveservertestcase

I have posted this to the django user google group/mailing list, but the message seems to be pending approval.
